### PR TITLE
chore(deps) - remove unused dependencies

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Remove unused dependencies from project:

| Package name | Added since | Unused since |
| --- | --- | --- | 
| `color.js` |  #4985 | #6798 |
| `url-parse` |  #2891 | #9915 |
| `vue-resize` |  #3739 | #4985 |
| `extendable-media-recorder-wav-encoder-worker` |  #5903 | shipped with `extendable-media-recorder-wav-encoder` |
| `workbox-webpack-plugin` |  #5903 | never used |
| `terser-webpack-plugin` |  #5903 | never used |
| `webpack-node-externals` |  #2185 | never used |

### 🚧 Tasks

- [ ] Manual testing (tested on prod build with web client + Chrome + Linux)

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not required
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,23 +33,20 @@
...
     "attachmediastream": "^2.1.0",
-    "color.js": "^1.2.0",
     "crypto-js": "^4.1.1",
     "debounce": "^1.2.1",
     "emoji-mart-vue-fast": "^15.0.0",
     "emoji-regex": "^10.2.1",
     "escape-html": "^1.0.3",
     "extendable-media-recorder": "^7.1.20",
     "extendable-media-recorder-wav-encoder": "^7.0.93",
-    "extendable-media-recorder-wav-encoder-worker": "^8.0.85",
```
Comment (nickvergessen) [inline package.json:44]: This is not removed from the deps and still used.
I assume we had to bind it to a specific version for some reason. Maybe @danxuliu remembers?

Comment (Antreesy) [reply package.json:44]: I don't mind to wait for it and resolve the question.

This is from my observation:
We are using `connect` function from this module: https://github.com/chrisguttandin/extendable-media-recorder-wav-encoder/blob/cd20c8b96fad327ef4b2aafad4a4d953755f577a/src/module.ts#L12
It uses minified version of worker package from its own dependencies: https://github.com/chrisguttandin/extendable-media-recorder-wav-encoder/blob/cd20c8b96fad327ef4b2aafad4a4d953755f577a/src/worker/worker.ts, https://github.com/search?q=repo%3Achrisguttandin%2Fextendable-media-recorder-wav-encoder%20extendable-media-recorder-wav-encoder-worker&type=code

Comment (ShGKme) [reply package.json:44]: It must be safe to remove.

We don't use it directly at all, it is only used by `extendable-media-recorder-wav-encoder`.
Moreover, the only work of `extendable-media-recorder-wav-encoder` is load end expose worker's methods.

And `extendable-media-recorder-wav-encoder` has the same version as listed here, so it doesn't affect resolved version at all.

**Review by ShGKme — APPROVED** [review]
Some details:
- `webpack-node-externals` is not used and usually useful only on nodejs backend development
- `terser-webpack-plugin` is used by `@nextcloud/webpack-vue-config`, but it can be safely removed as this is a direct dependency of Webpack (which is peer of `@nextcloud/webpack-vue-config` anyway)
- `workbox-webpack-plugin` is for **Service Workers** what we do not use

Also tested, that it works in general, including audio messages from files sidebar and calls.

File: package.json (hunk 1 above)
Comment (danxuliu) [reply package.json:44]: > This is not removed from the deps and still used. I assume we had to bind it to a specific version for some reason. Maybe @danxuliu remembers?

No idea, sorry, and the commit/pull request where [it was introduced does not say much either](https://github.com/nextcloud/spreed/commit/63fe1ac0d71cd660bee3df5ff3d304d5aca4871e#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R42) :shrug: But in any case I am late to the party and it was removed already, so... :-P

**Pull request merged**